### PR TITLE
Several GOG games tested

### PR DESCRIPTION
**Pull request by daddeltrotter**
I tested over the past month my complete GOG library.

### CODE REVIEW

File: gamedb.yaml [hunk 1]
```diff
@@ -5493,3 +5493,745 @@
...
+  id: 1090734724
+  banner: steam:243120
+  poster: https://cdn2.steamgriddb.com/grid/9fc497b1718cabc904e5af82c57bae53.png
+  background: https://cdn2.steamgriddb.com/hero/69a4241986721658eb6237d56c1b8abc.jpg
+  logo: https://cdn2.steamgriddb.com/logo/1145a30ff80745b56fb0cecf65305017.png
+  icon: https://cdn2.steamgriddb.com/icon/60c43e386965167ba97045a1fe957626.png
+  compat_tool: proton-stable
+  status: playable
+  notes:
+    - Displays kexboard icons
```
Comment (alkazar) [inline gamedb.yaml:5745]: Typo: keyboard

File: gamedb.yaml [hunk 2]
```diff
@@ -5493,3 +5493,745 @@
...
+
+- name: "Jazz Jackrabbit 2: The Secret Files"
+  platform: gog
+  id: 1351891846
+  banner: https://cdn2.steamgriddb.com/grid/5391707bb1c266d61d0c6d1e7948faaa.png
+  poster: https://cdn2.steamgriddb.com/grid/d6f993113df8012c1868e8612a279693.png
+  background: https://cdn2.steamgriddb.com/hero/8cff9bf6694dccfc3b6a613d05d51d16.png
+  logo: https://cdn2.steamgriddb.com/logo/54cf07b091f36d3e0a2d1e5bcf65d66a.png
+  icon: https://cdn2.steamgriddb.com/icon/d6d83fd1acd1918a677906afc9bdc1d7.ico
+  status: verified
```
Comment (alkazar) [inline gamedb.yaml:5815]: Is "verified" correct? The note says "limited controller support".

File: gamedb.yaml [hunk 3]
```diff
@@ -5493,3 +5493,745 @@
...
+    - No controller support
+
+- name: "Shadow Tactics: Blades of the Shogun"
+  platform: gog
+  id: 1601442230
+  banner: steam:418240
+  compat_tool: GE-Proton9-2
+  status: playable
+  notes:
+    - Enabling  controller with mouse
```
Comment (alkazar) [inline gamedb.yaml:5883]: What does this mean exactly? Also, there is an extraneous space.

File: gamedb.yaml [hunk 4]
```diff
@@ -5493,3 +5493,745 @@
...
+- name: "Venetica - Gold Edition"
+  platform: gog
+  id: 1345182195
+  banner: steam:338140
+  status: unsupported
+
+- name: "Worms Revolution Gold Edition"
+  platform: gog
+  id: 1421309312
+  banner: steam:200170
```
Comment (alkazar) [inline gamedb.yaml:6161]: The Steam version does not appear to be the gold edition.

Steamgriddb has some artwork for the gold edition:
https://cdn2.steamgriddb.com/grid/729fe11bfd2eab32e7613461d2f5ea74.png
https://cdn2.steamgriddb.com/grid/12059c268a74f3db7ed26b87efe23461.png

**Review by alkazar — CHANGES_REQUESTED** [review]
Thank you for this amazing work!

File: gamedb.yaml (hunk 1 above)
Comment (daddeltrotter) [reply gamedb.yaml]: corrected

File: gamedb.yaml (hunk 2 above)
Comment (daddeltrotter) [reply gamedb.yaml]: Sorry. Playable is correct.

File: gamedb.yaml (hunk 3 above)
Comment (daddeltrotter) [reply gamedb.yaml]: Game has not controller enabled on first start. Can be enabled in options.
I updated the wording herre.

File: gamedb.yaml (hunk 4 above)
Comment (daddeltrotter) [reply gamedb.yaml]: You're right. I missed this. Corrected.

**Review by alkazar — APPROVED** [review]

**Comment by alkazar** [comment]
Thank you for the updates.

**Pull request merged**